xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-macro/xwiki-platform-notifications-macro-ui/src/main/resources/XWiki/Notifications/Code/Macro/NotificationsMacro.xml

```diff
@@ -579,6 +579,10 @@ require(['jquery', 'XWikiNotificationsMacro'], function ($, XWikiNotificationsMa
   width: 2em;
 }
 
+.notification-icon > .img-thumbnail > .fa {
```

Comment:
This solution is not generic enough if the users use other Icon Themes, not just the Font Awesome.

Reply:
With Icon Themes, img are already well aligned because in style.css:
```
img {
    vertical-align: middle;
}
```